.github/workflows/build-jp5.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: setup workspace
-      run: ./setup_workspace.sh 5.0.2
+      run: yes | ./setup_workspace.sh 5.0.2
     - name: apply patches
-      run: ./apply_patches.sh apply 5.0.2
+      run: git config --global user.email "builder@example.com" && git config --global user.name "builder" && ./apply_patches.sh apply 5.0.2
     - name: build
       run: ./build_all.sh 5.0.2
```

Comment:
Did you see it fail if the build fail?
I am asking because we had GHA jobs that passed even if the bash return != 0.

Overall it looks great